labgrid/environment.py

```diff
@@ -33,7 +33,3 @@ def __attrs_post_init__(self):
     def get_target(self, role: str='main') -> Target:
         """Returns the specified target."""
         return self.targets[role]
-
-    def cleanup(self):
```

Comment:
This should stay, as the cleanup function on drivers should stay as well (see below).